go.mod

```diff
@@ -87,7 +87,7 @@ require (
 	github.com/mailru/easyjson v0.7.7 // indirect
 	github.com/mattn/go-isatty v0.0.16 // indirect
 	github.com/matttproud/golang_protobuf_extensions v1.0.2-0.20181231171920-c182affec369 // indirect
-	github.com/metal3-io/baremetal-operator/pkg/hardwareutils v0.1.1 // indirect
+	github.com/metal3-io/baremetal-operator/pkg/hardwareutils v0.1.2 // indirect
```

Comment:
this is not latest, I am seeing the same locally

Reply:
Good Catch! fixed

Reply:
and it should be coming from https://github.com/metal3-io/baremetal-operator/blob/v0.2.0/apis/go.mod#L7?

Reply:
Yes you are right :(

Reply:
but as I have uplifted here, it should still be fine, BMO issue has to be solved separately.